english/virtualreality/voice/strangeVoice.md

```diff
@@ -13,4 +13,19 @@ and your right hand looks like a hotdog.
 Suddenly you feel very tired and you're starting to fall
 alseep.
 
-Just before falling asleep, you wonder where you'll wake up...
\ No newline at end of file
+Just before falling asleep, you wonder where you'll wake up...
+
+You dream that you are a drop of water, falling through the air, 
+warping and shedding small pieces of yourself as you are buffeted by the air currents.
+
+The other droplets around you seem content to fall as preordained by physical forces of the world. 
+Perhaps they once fought to control their destiny as well, but today they were firmly entrenched in the water cycle.
+ Falling from tens of thousands of feet to the ground was an ordinary Saturday Morning. 
+
+But this was different for you, a first rain, you only have one of those. 
+You peer down below you and see you have two choices, left or right. The Fish Sanctuary or The pig farm?
+
+[The Pig Farm](../../water-dreams/pig-farm/pig-farm.md)
+
+[The Fish Sanctuary](../../water-dreams/fish-sanctuary/fish-sanctuary.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.